packages/presenters/test/test.ts

```diff
@@ -48,7 +48,7 @@ describe('Presenters', () => {
       ];
       assert.deepEqual(removeSourceMap(await bundle(files)), [{
         name: 'bundle.js',
-        text: "(function () {\n\t'use strict';\n\n\tconst foo = \"bar\";\n\n\tconsole.log(foo);\n\n}());\n"
+        text: "(function () {\n\t'use strict';\n\n\tvar foo = \"bar\";\n\n\tconsole.log(foo);\n\n}());\n"
```

Comment:
Thanks @micahstubbs for having a look! This change is because now with [Buble](https://github.com/Rich-Harris/buble) in the pipeline, all ES6 code is transpiled to ES5. So, all `const` and `let` are transpiled to `var`.

I adopted Buble mainly for its JSX support. The ES5 transpilation is sort of a win, but honestly it's not really required.

Was also investigating [https://www.npmjs.com/package/rollup-plugin-jsx](https://www.npmjs.com/package/rollup-plugin-jsx), which provides _only_ JSX, but its last release was 3 years ago.

I wonder if there's a way to use Buble for _just_ JSX, omitting the ES5 transpilation (and hopefully reducing bundle size as well). That would be ideal, but not sure if it's possible.

Still open to various alternatives. Buble seemed the best fit but there may be other options.